docs/guide/contributing.rst

```diff
@@ -71,11 +69,28 @@ Prerequisites
    1. In macOS: Open the **Terminal** app.
    2. In Windows: From the Start Menu, open **Git Bash**.
 
-4. In the repo UI, find the file name corresponding to an `existing issue`_ or page you want to improve. 
+4. In the `Write the Docs www project`_, find the file name corresponding to an `existing issue`_ or page you want to improve. 
 
 For example, ``/docs/documentarians.rst`` creates https://www.writethedocs.org/documentarians/. 
 
-5. Review formatting guidelines for the Markup style the file uses. The example file above uses `reStructuredText (.rst) Markup`_. Other Markup styles for our guides include Markdown (.md).
+5. Review formatting guidelines for the Markup style the file uses. The example file above uses `reStructuredText (.rst)`_.
+
+.. _update-guide-via-github:
+
+Editing a single guide directly in the GitHub UI
+--------------------------------------------------
+
+1. :ref:`Complete the prerequisites <prerequisites>`.
```

Comment:
I see a couple problems with this:
* These aren't requirements for using the UI (only for the terminal)
* I don't think the first step in a guide should ever be "complete the prerequisites". I think that should be clear enough from the general structure.
Only the first is really a blocking issue.